app/src/main/java/swati4star/createpdf/util/FileUtils.java

```diff
@@ -12,6 +13,7 @@
 import android.print.PrintDocumentAdapter;
 import android.print.PrintManager;
 import android.provider.MediaStore;
+
```

Comment:
[I repeat] Any beautification work should be done in separate commit may or may not in single pull request.